src/Network/NWFramer.cs

```diff
@@ -0,0 +1,474 @@
+//
+// NWFramer.cs: Bindings the Network nw_framer_t API.
+//
+// Authors:
+//   Manuel de la Pena (mandel@microsoft.com)
+//
+// Copyrigh 2019 Microsoft Inc
+//
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+using ObjCRuntime;
+using Foundation;
+using CoreFoundation;
+
+using OS_nw_framer=System.IntPtr;
+using OS_nw_protocol_metadata=System.IntPtr;
+using OS_dispatch_data=System.IntPtr;
+using OS_nw_protocol_definition=System.IntPtr;
+using OS_nw_protocol_options=System.IntPtr;
+using OS_nw_endpoint=System.IntPtr;
+using OS_nw_parameters=System.IntPtr;
+
+namespace Network {
+
+	[TV (13,0), Mac (10,15), iOS (13,0), Watch (6,0)]
+	public class NWFramer : NWProtocolMetadata {
```

Comment:
@migueldeicaza I'm curious about this inheritance, looking at the headers, and the properties expose by the NWProtocolMetadata object my understanding is that NWProtocolMetadata is the parent class.